python/python/Ice/BatchRequestInterceptor.py

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) ZeroC, Inc.
 
-class BatchRequestInterceptor(object):
+class BatchRequestInterceptor:
```

Comment:
This is not required with Python 3.x, all classes are implicitly derived from object.